### ISSUE TITLE
Image Link for the Tutorial don't work.

### ISSUE THREAD

**Issue by simonhellweger** [angular/angular#58144]
### Describe the problem that you experienced

I was doing the Angular Tutorial. I wondered why the images won't appear. I checked the code multiple times. After some time i checked the url. Then I saw the link was invalid.

### Enter the URL of the topic with the problem

https://angular.dev/assets/images/tutorials/common

### Describe what you want to experience that would fix the problem

Fix the link.

### Add a screenshot if that helps illustrate the problem

![Image](https://github.com/user-attachments/assets/0e385e57-62f1-4d58-8c6c-68b0f4a916b1)

**Comment by geromegrignon** [comment]
Hi @simonhellweger,

The link is introduced in this tutorial step (https://angular.dev/tutorials/first-app/04-interfaces).
As explained by snippet codes, your link is not used directly, as being the folder hosting the images:

```ts
export class HomeComponent {
  readonly baseUrl = 'https://angular.dev/assets/images/tutorials/common';
  housingLocation: HousingLocation = {
    id: 9999,
    name: 'Test Home',
    city: 'Test city',
    state: 'ST',
    photo: `${this.baseUrl}/example-house.jpg`,
    availableUnits: 99,
    wifi: true,
    laundry: false,
  };
}
```

Is it related to this step or another one?
If so please add details to pinpoint the issue.

**Comment by simonhellweger** [comment]
Yes it is related to this step. I copied the code from the tutorial however the links in "photo: `${this.baseUrl} (link)" don't display anything. However, at the step where we include the database, those links work again.

**Comment by vevecna** [comment]
hello, you have passed to the photo property on this link array [Array list of homes](https://gist.github.com/MarkTechson/efe8a9d4727ef33949b78812e66db082) with the exactly interpolation string? 
Example with the home index 1:  

```
housingLocationList: HousingLocationInfo[] = [
    {
      id: 0,
      name: 'Acme Fresh Start Housing',
      city: 'Chicago',
      state: 'IL',
      photo: `${this.baseUrl}/bernard-hermant-CLKGGwIBTaY-unsplash.jpg`, //Here are the link that works for me
      availableUnits: 4,
      wifi: true,
      laundry: true
    },
. . .
```

**Comment by josephperrott** [comment]
Closing as after going through the tutorial myself to confirm, It appears to work as expected. 

While we do show that the baseURL for where are images are hosted is at `https://angular.dev/assets/images/tutorials/common`. As noted in instructions, this is used as the base url to which the specific photo url is constructed by adding the filename to this base url.  While this base url does not serve a specific asset.  The url for the first photo described in the tutorial works as expected at: `https://angular.dev/assets/images/tutorials/common/example-house.jpg`
